### PR TITLE
feat: /data/kafka/what-is-kafka build

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Built page as per [doc](https://docs.google.com/document/d/1IThZcfKZcv-ArqU_0n8A181AGkQPUn_Iq1O4sqk5GTs/edit?tab=t.0) and [design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=2135-30990&node-type=instance&m=dev) 

## QA

- View the site locally in your web browser at: https://canonical-com-1408.demos.haus/data/kafka/what-is-kafka
- Check against doc and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-15054

### CODE REVIEW

**Comment by mattea-turic** [comment]
mainly just padding changes :)

- could you add a `<br>` after "modern" in the "The benefits of Kafka" section pls
- "how do companies use kafka" should have shallow bottom padding before the feature block
- same thing for the "how does kafka work" section, for the image before the next paragraph
- and also for the image in "installing kafka"
- finally, shallow padding before the link for each of the items in "canonical's charmed kafka"
Screenshot 2024-10-25 at 11 03 30
Screenshot 2024-10-25 at 11 05 50

File: templates/data/kafka/what-is-kafka.html [hunk 1]
```diff
@@ -0,0 +1,566 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1bfo97XdFdKlzrMcMoRxiTdzt_bPVndx6j55IPsapuAE/edit
```
Comment (britneywwc) [inline templates/data/kafka/what-is-kafka.html:4]:
```suggestion
  https://docs.google.com/document/d/1IThZcfKZcv-ArqU_0n8A181AGkQPUn_Iq1O4sqk5GTs/edit?tab=t.0
```

File: templates/data/kafka/what-is-kafka.html [hunk 2]
```diff
@@ -0,0 +1,566 @@
...
+      <div class="col">
+        <p>
+          Apache Kafka includes mature information security controls like TLS wire encryption, authentication and authorisation and access logging.
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50 p-section--shallow">
```
Comment (britneywwc) [inline templates/data/kafka/what-is-kafka.html:409]: Can you separate the structural classes into two different divs please for better coding practices

File: templates/data/kafka/what-is-kafka.html [hunk 3]
```diff
@@ -0,0 +1,566 @@
...
+          <a class="p-button--positive"
+             href="https://ubuntu.com/engage/kafka-reference-architecture">Download the Kafka reference architecture guide</a>
+          <a href="https://ubuntu.com/blog/what-is-apache-kafka">Learn more more about Apache Kafka&nbsp;&rsaquo;</a>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50 p-section--shallow">
```
Comment (britneywwc) [inline templates/data/kafka/what-is-kafka.html:52]: Similarly here, can the row and section classes be two different divs?

File: templates/data/kafka/what-is-kafka.html (hunk 2 above)
Comment (mtruj013) [reply templates/data/kafka/what-is-kafka.html:409]: Yes, thanks for catching!

**Review by britneywwc — APPROVED** [review]
Looks great, thanks for making the changes! :)

**Pull request merged**